webapp/components/ContributionForm/ContributionForm.vue

```diff
@@ -7,11 +7,22 @@
     @submit="submit"
   >
     <template slot-scope="{ errors }">
+      <base-button
+        v-if="showDeleteImageButton"
+        class="delete-image"
+        icon="close"
+        size="small"
+        circle
+        danger
+        filled
+        @click.prevent="deleteImage"
+      />
       <hc-teaser-image
         :contribution="contribution"
-        @addTeaserImage="addTeaserImage"
         :class="{ '--blur-image': form.blurImage }"
+        @addTeaserImage="addTeaserImage"
         @addImageAspectRatio="addImageAspectRatio"
+        @cropInProgress="cropInProgress"
```

Comment:
> [<img alt="mattwr18" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/mattwr18) **Authored by [mattwr18](https://github.com/mattwr18)**
_<time datetime="2020-01-16T14:41:14Z" title="Thursday, January 16th 2020, 3:41:14 pm +01:00">Jan 16, 2020</time>_

---
> **Outdated (history rewrite)** - original diff
---
```diff
@@ -16,7 +23,7 @@
         <img
           v-if="contribution"
           class="contribution-image"
-          :src="contribution.image | proxyApiUrl"
+          :src="contribution.image || proxyApiUrl"
```


```suggestion
          :src="contribution.image | proxyApiUrl"
```

Reply:
> [<img alt="mattwr18" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/mattwr18) **Authored by [mattwr18](https://github.com/mattwr18)**
_<time datetime="2020-01-16T14:44:09Z" title="Thursday, January 16th 2020, 3:44:09 pm +01:00">Jan 16, 2020</time>_

---


this single pipe character `|` runs `contribution.image` through a filter at `webapp/plugins/vue-filters.js`
which takes in a url(contribution.image) and if it starts with a `/` meaning it's a relative url, then it replaces `/` with `/api/` so that it proxies our backend for the image
double pipes `||` mean `or`, maybe you know all this and it's an unintended change, but it's not one we want, I guess.